apps/spark/build.gradle

```diff
@@ -68,8 +68,13 @@ dependencies {
     implementation 'com.fasterxml.woodstox:woodstox-core:6.2.7'
 
     // open telemetry related classed. Latest Okhttp version is 4.10.0, pinning to 4.9.3 to avoid dependency issues
```

Comment:
shall we move this comment to where `ok_http3_version` is defined

Reply:
Fixed